frontend/src/pages/Settings/settingsArr.css

```
@@ -942,15 +961,15 @@
 
 .arr-table__path {
   display: block;
-  color: var(--aurral-gray-light);
+  color: var(--aurral-text-muted);
   font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, monospace;
   font-size: 0.8125rem;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)